unicodetools/org/unicode/propstest/ListProps.java

```diff
@@ -41,6 +41,7 @@
 
     public static void main(String[] args) {
         IndexUnicodeProperties latest = IndexUnicodeProperties.make();
+        System.out.println("UCD: " + latest.getUcdVersion());
```

Comment:
totally optional: prefix the output with the name of the class ListProps?

Reply:
```suggestion
        System.out.println(ListProps.class.getName() + ": UCD: " + latest.getUcdVersion());
```

like that?

Reply:
lg